nishimweelysee/Elysee-CapstoneProject
# Ft update article

#### What does this PR do?
> This PR contains a branch for Updating an existing article.
>When an article created, you may send it with incorrect information and you need to update it.
>This will help to update that article either Blog or Portfolio project. 
#### Description of Task to be completed
>
>- update button to save changes
>
#### How should this be manually tested?
>1. Open a website
>2. go to admin nav
>3. go to blog or portfolio tab
>4. you will see an update button
>5. you will a search button also you may search it and get information then update it.
>6. or you may see on list and and select which one you need to update.
#### Any background context you want to provide?
>
#### What are the relevant Trello story?
> Ft(Update an existing article) use this link https://trello.com/c/mXgq4czO to visiti this trello card.
#### Screenshots (if appropriate)
>
> This is for update blog
>
<img width="363" alt="update blog" src="https://user-images.githubusercontent.com/56052588/92358299-979e4680-f0e9-11ea-968a-e8f67eed342b.PNG">
>
> This for update Portfolio
>
<img width="361" alt="update portfolio" src="https://user-images.githubusercontent.com/56052588/92358302-98cf7380-f0e9-11ea-8e69-5d9b0f9fe254.PNG">
>
>
> Thank you !!
